libs/coin-framework/src/bot/specs.ts

```diff
@@ -94,10 +95,6 @@ export function deviceActionFlow<T extends TransactionCommon>(
   description: FlowDesc<T>
 ): DeviceAction<T, State<T>> {
   return (arg: DeviceActionArg<T, State<T>>) => {
-    invariant(
-      arg.appCandidate.model === "nanoS",
```

Comment:
I don't think nanoX is supported though, maybe keep a condition on this.

Reply:
I can try with nanoX but. I think it's the same OCR issue ?